interfaces/builtin/optical_drive.go

```diff
@@ -46,13 +46,18 @@ const opticalDriveConnectedPlugAppArmor = `
 # Allow read access to optical drives
 /dev/sr[0-9]* r,
 /dev/scd[0-9]* r,
+/dev/sg[0-9]* r,
```

Comment:
This isn't necessarily just optical drives, so adding a clarifying comment would be good:
```
# allow all generic scsi devices here and use the device cgroup to
# differentiate optical drives
/dev/sg[0-9]* r,
```